src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import './util/variables';
+@import './blocks/card';
+@import './blocks/stars';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
```

Comment:
Consider moving general styling to a separate file to keep your main file clean and clear